bundle/edu.gemini.spModel.io/src/test/scala/edu/gemini/spModel/io/impl/migration/to2016B/TargetMigrationTest.scala

```diff
@@ -102,7 +102,7 @@ class TargetMigrationTest extends Specification with MigrationTest {
     "Migrate MPC Minor Planets" in withTestProgram2("targetMigration.xml") { p =>
       p.findBaseByObsTitle("mpc-minor-planet") must_== Some(NonSiderealTarget(
         "beer",
-        IMap(1457994485000L -> Coordinates.fromDegrees(88.769, 21.08299999999997).get),
+        Ephemeris(Site.GN, IMap(1457994485000L -> Coordinates.fromDegrees(88.769, 21.08299999999997).get)),
```

Comment:
I assume the "I" in `IMap` stands for invariant and I'm guessing it is providing an option that looks more natural if you instantiate the type with standard prefix notation `IMap[A, B]` as opposed to infix `A ==>> B`.